src/handlers/utils.js

```diff
@@ -18,10 +19,22 @@ export function createPostbackAction(label, input, issuedAt) {
  * @return {string} Description of feedback counts
  */
 export function createFeedbackWords(positive, negative) {
-  if (positive + negative === 0) return '[還沒有人針對此回應評價]';
+  if (positive + negative === 0) return `[${t`No feedback yet`}]`;
   let result = '';
-  if (positive) result += `有 ${positive} 人覺得此回應有幫助\n`;
-  if (negative) result += `有 ${negative} 人覺得此回應沒幫助\n`;
+  if (positive)
+    result +=
+      ngettext(
+        msgid`${positive} user consider this helpful`,
```

Comment:
```suggestion
        msgid`${positive} user considers this helpful`,
```
and a few lines below as well